web/docs/advanced/web-sockets.md

```diff
@@ -222,7 +222,7 @@ export const ChatPage = () => {
 </TabItem>
 <TabItem value="ts" label="TypeScript">
 
-Wasp's **full-stack type safety** kicks in here: all the event types and payloads are automatically inferred from the server and are available on the client 🔥
+Wasp's **full-stack type safety** kicks in here: all the event types and payloads are automatically inferred from the server and are available on the client.
```

Comment:
We're lucky I found this one in time before the emoji police caught wind of it.

Reply:
Or was this a planted piece of emoji to test us?